src/components/AccessibilityPolicyForm/AccessibilityPolicyForm.test.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import Enzyme from 'enzyme';
-import { AccessibilityPolicyForm } from './index';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+
+import WrappedAccessibilityPolicyForm, { AccessibilityPolicyForm } from './index';
```

Comment:
So, does this mean we should start using the redux-mock-store library to fake the redux store instead of exporting the unwrapped component from now on? If so, I think we should put a ticket in the backlog or do something to change the other tests to match so that we don't follow the old pattern from other test files.